axum-extra/Cargo.toml

```diff
@@ -34,7 +34,7 @@ spa = ["tower-http/fs"]
 typed-routing = ["dep:axum-macros", "dep:serde", "dep:percent-encoding"]
 
 [dependencies]
-axum = { path = "../axum", version = "0.6.0-rc.1", default-features = false }
+axum = { path = "../axum", version = "0.6.0-rc.2", default-features = false }
```

Comment:
I think we should do one of two things here: Either add an exact version dependency so future rc's or the full axum 0.6.0 don't get used as a dependency of axum-extra 0.4.0-rc.2; or not release another rc of axum-extra at all (it doesn't seem to be required, right?).

Reply:
I assumed it was required but if axum-extra-rc.1 can use axum-rc.2 then yeah we don't need a release of axum-extra. I assumed rc versions were considered incompatible.

Reply:
Yeah they really should be, but aren't. That's why adding the `=` to the axum-core dependency is important.

Reply:
Alright I've reverted the new version of axum-extra.